app/controllers/order_items_controller.rb

```diff
@@ -8,19 +8,19 @@ def create
     result = CreateOrderItem.call(params: order_item_params, current_order: current_order)
 
     if result.success?
-      redirect_to order_order_items_path, notice: I18n.t('order.added')
+      redirect_to order_order_items_path(order_id: current_order.id), notice: I18n.t('order.added')
     else
-      redirect_to order_order_items_path, alert: I18n.t('order.wrong')
+      redirect_to order_order_items_path(order_id: current_order.id), alert: I18n.t('order.wrong')
```

Comment:
Is it possible to pass just an object?
`redirect_to order_order_items_path(order: current_order), alert: I18n.t('order.wrong')`